src/addRemoveItem.js

```diff
@@ -0,0 +1,29 @@
+const removeItem = (index, list) => {
+  list.splice(index, 1);
+
+  for (let i = 0; i < list.length; i += 1) {
+    list[i].number = i;
+  }
+
+  return list;
+};
+
+const addItem = (list, description) => {
+  const item = {
+    description,
+    completed: false,
+    number: list.length,
+  };
```

Comment:
- [ ] It would be so much better if your index in your case number is set to the value of the new array length. As is one of the requirements.

![Capture](https://user-images.githubusercontent.com/64173189/134612586-435129da-4cbe-42a2-8800-4525b9bb1767.JPG)

 Currently, your number starts from `0` and it must start from `1`. 

You could fix this minor issue by just adding a `+1` to the `list.length`

```suggestion
const addItem = (list, description) => {
  const item = {
    description,
    completed: false,
    number: list.length +1 ,
  };
```

👍 💯